kobo/worker/logger.py

```diff
@@ -31,6 +32,7 @@ def __init__(self, hub, task_id, *args, **kwargs):
         self._running = True
         self._send_time = 0
         self._send_data = b""
+        self._timeout = int(os.environ.get("KOBO_LOGGING_THREAD_TIMEOUT", 120))
```

Comment:
Minor: I feel like this default should probably be a bit longer.

Our max availability target for relevant services will be 99.5% which means a service is "allowed" to be down 0.5% of the time before any action must be taken.  This works out to about ~7 minutes of "allowed" downtime per day.

Thinking about it this way, I would be more comfortable setting the default to around 10 minutes so that the logic is designed for tasks to survive an outage if the outage is resolved within the 99.5% availability target.